src/components/Questions/TopQuestions.vue

```diff
@@ -0,0 +1,69 @@
+<template>
+  <div>
+    <div v-if="loading" class="text-xs-center">
+      <v-progress-circular
+        :size="50"
+        color="primary"
+        indeterminate
+      ></v-progress-circular>
```

Comment:
you can write `<foo />` instead of `<foo></foo>`

Reply:
Ok, thanks)